### DISCUSSION TITLE
installazione HA OS Mini PC

### DISCUSSION THREAD

**Discussion by dibummer** [indomus/forum#1846] [HUB: Home Assistant]
Ciao,
cerco di installare hA OS su un mini pc, seguo quanto indicato nella guida di indomus;
ho flashato l'immagine, predisposto l'avvio del pc da usb;
quando inizia non mi appare la live di Ubuntu per procedere all'installazione di HA, di fatto procede alla configurazione di HA sino ad arrivare alla schermata del terminale.
dove sbaglio?
grazie

**Comment by NdR91** [comment]
C'è un pò di confusione. Stai installando HA OS o Ubuntu?

**Reply by dibummer** [reply]
HA OS

**Reply by NdR91** [reply]
Allora non ho capito il nesso con Ubuntu...

**Reply by dibummer** [reply]
ho risolto.
si trattava del file di installazione di HA OS che avevo scaricato dal sito indomus, che probabilmente avevo scaricato male;
in sostanza nel boot non installava la live di ubuntu per poi installare sul disco HA OS da riga di comando;
quindi, ho scaricato nuovamente il file, caricato su una Pen drive con balenaEtcher ed ha funzionato sono riuscito ad installare HA OS sul SSD del mini pc.
adesso devo ripristinare HA da un backup che avevo sul Qnap in virtuale.
grazie dell'intervento.
